aten/src/ATen/native/native_functions.yaml

```diff
@@ -10492,11 +10510,29 @@
 - func: linalg_matrix_power.out(Tensor self, int n, *, Tensor(a!) out) -> Tensor(a!)
   python_module: linalg
 
-- func: linalg_matrix_rank(Tensor self, float? tol=None, bool hermitian=False) -> Tensor
+- func: linalg_matrix_rank.atol_rtol_tensor(Tensor input, *, Tensor? atol=None, Tensor? rtol=None, bool hermitian=False) -> Tensor
+  python_module: linalg
```

Comment:
See above FC concern.

This may require offline discussion for how to resolve. I've added it to our linalg meeting schedule.